qemu/tests/cfg/virtio_fs_nfs_migration.cfg

```diff
@@ -0,0 +1,172 @@
+        viofs_log_enable_cmd = 'reg add HKLM\Software\VirtIO-FS /v DebugLogFile /d ${viofs_log_file} /t REG_SZ'
+        viofs_debug_delete_cmd = 'reg delete HKLM\Software\VirtIO-FS /v DebugFlags /f'
+        viofs_log_delete_cmd = 'reg delete HKLM\Software\VirtIO-FS /v DebugLogFile /f'
+        viofs_reg_query_cmd = 'reg query HKLM\Software\VirtIO-FS'
+        virtio_win_media_type = iso
+        cdroms += " virtio"
+        cmd_md5 = "%s: && md5sum.exe ${test_file}"
+        cmd_dd = 'dd if=/dev/random of=%s bs=1M count=100'
+        guest_file = "%s:\${test_file}"
+        fio_options = '--name=${fio_name} --filename=%s --ioengine=windowsaio'
+        fio_options += ' --rw=rw --bs=1k --size=200M --numjobs=${numjobs} --runtime=1800 --time_based --thread'
+        fio_name = fio.exe
+        cmd_chk_fio = 'TASKLIST /FI "IMAGENAME eq FIO.EXE'
+        fio_filename = "%s\:\${test_file}.fio"
+    variants:
+        - cache_mode_auto:
+            fs_binary_extra_options += " --cache auto"
+            fs_binary_extra_options_targetfs += " --cache auto"
+        - cache_mode_always:
+            fs_binary_extra_options += " --cache always"
+            fs_binary_extra_options_targetfs += " --cache always"
+        - cache_mode_never:
+            fs_binary_extra_options += " --cache never"
+            fs_binary_extra_options_targetfs += " --cache never"
+        - cache_mode_metadata:
+            fs_binary_extra_options += " --cache metadata"
+            fs_binary_extra_options_targetfs += " --cache metadata"
+    variants:
+        - @default:
+        - writeback:
+            only cache_mode_auto cache_mode_always
+            migrate_parameters = "{'downtime-limit': 30000}"
+            fs_binary_extra_options += " --writeback"
+            fs_binary_extra_options_targetfs += " --writeback"
+        - dio:
+            only cache_mode_never
+            fs_binary_extra_options += " --allow-direct-io"
+            fs_binary_extra_options_targetfs += " --allow-direct-io"
+    variants:
+        - find_paths:
+        - file_handles:
+            required_virtiofsd_version = [1.13.0,)
+            virtiofsd_version_cmd = rpm -q virtiofsd | cut -d- -f2
+            fs_binary_extra_options_targetfs = " --modcaps=+dac_read_search"
+            variants:
+                - never:
+                    fs_binary_extra_options += " --inode-file-handles=never --migration-mode=file-handles --modcaps=+dac_read_search"
+                - mandatory:
+                    fs_binary_extra_options += " --inode-file-handles=mandatory --migration-mode=file-handles --modcaps=+dac_read_search"
+    variants:
+        - @default:
+        - multifs:
+            only find_paths.default.cache_mode_auto
+
+            filesystems += " fs2"
+            filesystems_migration += " targetfs2"
+            fs_binary_extra_options_targetfs2 = " --cache auto --modcaps=+dac_read_search"
```

Comment:
Hi, as we are going to test find_paths only so here we do not need `--modcaps=+dac_read_search`, which is used for file_handles mode.

Reply:
removed it, change done